streamingcli/jupyter/notebook_converter.py

```diff
@@ -201,6 +201,10 @@ def _render_flink_app(notebook_entries: Sequence[NotebookEntry]) -> ConvertedNot
         return ConvertedNotebook(content=autopep8.fix_code(flink_app_script), remote_jars=list(remote_jars),
                                  local_jars=list(local_jars))
 
+    @staticmethod
+    def _skip_statement(statement: str) -> bool:
+        return statement.lower().startswith(('select', 'show', 'desc'))
```

Comment:
I would also add `DESCRIBE` and `EXPLAIN`

Reply:
Can we use is_dql from magics ?

Reply:
I think is_dql is a bit too complex for this case. We would have to declare extra dicts and a method to have the same logic in more than 1 line.

Reply:
Hmmm. With your solution this statement will be included (whitespace at the beginning)
```
    SELECT xxx...
```